### PR TITLE
Updated copy for attributes and events

### PR DESCRIPTION
**Pull request by emilyyidan**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -62,18 +63,17 @@ export type BentoSettings = {
...
   accountUser: {
     /**
      * Unique identifier for the account user
-     * This is best set to something used internally by your app
+     * This is best set to something used internally by your app like an id.
      */
     id: string;
     /**
-     * (RECOMMENDED) Human-readable unique account user identifier
+     * (Recommended, but not required) Human-readable unique account user identifier
```
Comment (jpcbarros) [inline src/index.ts:76]: nit: Whether or not required is specified by the types themselves, so I considered it redundant to mention it in the description as well.

In this case, the `?` right after `fullName` indicates to Typescript that this property is optional

File: src/index.ts [hunk 2]
```diff
@@ -93,15 +93,15 @@ export enum BentoWindowEvents {
...
+  /** Fired when a Guide is loaded */
   onGuideLoad = "bento-onGuideLoad",
-  /** You can fire this event to tell Bento to close the sidebar component */
+  /** You can fire this event to tell Bento to close the Sidebar component */
   sidebarClose = "bento-sidebarClose",
-  /** You can fire this event to tell Bento to open the sidebar component, when available */
+  /** You can fire this event to tell Bento to open the Sidebar component, when available */
   sidebarOpen = "bento-sidebarOpen",
-  /** Fired when the Inline component is loaded */
+  /** Fired when the Inline (i.e. embedded card or checklist) component is loaded */
```
Comment (jpcbarros) [inline src/index.ts:104]: This isn't true for embedded cards. For those, we fire an event containing the Guide Entity ID, therefore as far as I know it is impossible to add the types here. 

See #8345 for more details

**Review by jpcbarros — COMMENTED** [review]
Looks good, but left 2 small comments

File: src/index.ts (hunk 1 above)
Comment (emilyyidan) [reply src/index.ts:76]: cool, I'll leave it to you. Decided to clarify bc we get asked so many questions about this.

File: src/index.ts (hunk 2 above)
Comment (emilyyidan) [reply src/index.ts:104]: Cool, will leave it to you to modify. in my experience no one understands what "inline" means so clarifying we mean the onboarding checklist is likely helpful.

**Pull request merged**